src/index.js

```diff
@@ -1,5 +1,5 @@
-import redata from './redata';
+import configRedata from './redata';
 import reactRedata from './react/react-redata';
 
-export default redata;
+export default configRedata;
```

Comment:
Hmm perhaps we can revisit this later. I have not yet read the codebase yet but we can improve the naming of things later on.